packages/sdk/src/utils/index.ts

```diff
@@ -18,7 +18,10 @@ import {
 
 export function getNetwork(value: Network) {
   if (value === "mainnet") {
-    return bitcoin.networks["bitcoin"]
+    return bitcoin.networks.bitcoin
+  }
```

Comment:
should we add testnet and regtest here too?

Reply:
testnet and regtest already accounted for in the default path `bitcoin.networks[value]` only mainnet key is `bitcoin` instead of `mainnet` and no regtest

Reply:
oh right I didn't see the default path